src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -265,8 +297,21 @@ public function getPath()
             ));
         }
 
-        $this->validateType($leftSide);
-        $this->validateType($rightSide);
+        if ($leftPlaceholder = (is_string($leftSide) && isset(self::$placeholders[$leftSide]))) {
+            $this->validateType(self::$placeholders[$leftSide]);
+        } else {
+            $this->validateType($leftSide);
+        }
+
+        if ($rightPlaceholder = (is_string($rightSide) && isset(self::$placeholders[$rightSide]))) {
+            $this->validateType(self::$placeholders[$rightSide]);
+        } else {
+            $this->validateType($rightSide);
+        }
+
+        if ($leftPlaceholder || $rightPlaceholder) {
+            return $rightSide;
```

Comment:
probably the same as calling `mergeValues` but i dont wanna pass a placeholder, while we validated a different value.